catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -725,7 +728,7 @@ protected InputTransformer lookupTransformerReference(String namespaceUri)
   }
 
   @VisibleForTesting
-  Bundle getBundle() {
+  protected Bundle getBundle() {
     return FrameworkUtil.getBundle(this.getClass());
```

Comment:
I would strongly suggest to actually supply the service references you need in the constructor instead of using the static `FrameworkUtil` as its hard to test (as evidenced by the annotation on the method) and it hides what you actually need.